docs/src/man/parallel_approximations.md

```diff
@@ -0,0 +1,145 @@
+# Parallel Approximations
+
+A subset of the approximation algorithms are implemented in parallel in the 
+`LazySets.Parallel` module. In order to use parallel versions of the algorithms,
+you can write:
+
+```@example
+using LazySets
+import LazySets.Parallel
+
+# call a method implemented in parallel, for example:
+S = Ball2(ones(100), 1.0)
+Parallel.box_approximation(S)
+```
+
+Note that after importing or using `LazySets.Parallel`, the version of the function
+used must be fully qualified, eg. `LazySets.Approximations.box_approximation` for the
+sequential function or `LazySets.Parallel.box_approximation` for the parallel version.
+
+The parallelization strategy that is available uses processes. To set the number
+of processes use the flag `-p N` at julia startup, where `N` is the number of
+desired processes. For example, do
+
+```julia
+$ julia -p 4
+```
+to launch `4` additional local worker julia processes. Use the keyword `auto`,
+as in
+```julia
+$ julia -p auto
+```
+to launch as many workers as the number of local CPU threads.
```

Comment:
```suggestion
to launch as many workers as the number of local CPU cores.
```
I saw that "CPU threads" is used in the Julia manual, but I find it wrong.

Reply:
in `$julia -h` it says

```julia
 -p, --procs {N|auto}      Integer value N launches N additional local worker processes
                           "auto" launches as many workers as the number of local CPU threads (logical cores)
```

info for the local machine is got with `Base.Sys.CPU_THREADS` and
and with `InteractiveUtils.versioninfo(verbose=true)`

Reply:
Okay, [it is still wrong](https://en.wikipedia.org/wiki/Thread_(computing)), but truth is relative :smile:

Reply:
that article mentions logical cores here:

> The kernel can assign one thread to each logical core in a system (because each processor splits itself up into multiple logical cores if it supports multithreading, or only supports one logical core per physical core if it does not), and can swap out threads that get blocked. However, kernel threads take much longer than user threads to be swapped. 

in other words, there is a distinction between `local CPU cores` and `logical CPU cores`.

Reply:
Incidentally, there's a recent Discourse thread on this topic:
https://discourse.julialang.org/t/parallel-good-practice/17486/7